### PR TITLE
Don't overshoot the viewport

### LINKED ISSUE #316: Width stretches past actual viewport
See the top RHS.

screen shot 2016-03-16 at 15 58 43

### PR DESCRIPTION
**Pull request by lotrfan**
Closes #316

### CODE REVIEW

**Comment by lotrfan** [comment]
Probably easier to review here: https://github.com/PrairieLearn/PrairieLearn/pull/317/files?w=1

or with `--ignore-space-change` on the commandline

File: frontend/require/NavView.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ define(['underscore', 'backbone', 'mustache', 'text!NavView.html'], function(_,
 
     var NavView = Backbone.View.extend({
 
-        tagName: 'nav',
+        tagName: 'div',
```
Comment (mwest1066) [inline frontend/require/NavView.js]: I don't understand why this is changed? When I tested, it doesn't seem to make any difference either way?

Comment (lotrfan) [inline frontend/require/NavView.js]: It probably doesn't matter... it just seemed weird to have a `<nav>` nested in a `<nav>`

Comment (mwest1066) [inline frontend/require/NavView.js]: Ah, I get it now! I was thinking of the `tagName: 'nav'` as being the same thing as the top-level `<nav>` in `NavView.html`, but of course the tag will actually be a new enclosing element. Thanks for explaining!

**Pull request merged**